### ISSUE TITLE
Subsequent quote lines are rendered as blockquotes

### ISSUE THREAD

**Issue by alexwennerberg** [skyjake/lagrange#108]
I noticed that Lagrange handles subsequent quote lines as a singe block. Here's an example:

```
> First line
> Second line
> third line

Some other text 

> Single line: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus et efficitur mi. Vivamus enim risus, sagittis a magna a, faucibus ultrices erat. Mauris a imperdiet ex. Suspendisse mattis urna mi, in sodales dolor pellentesque at. Etiam malesuada erat at tortor mattis lacinia. Nunc lacinia auctor sem eu suscipit. Aliquam sollicitudin nibh lacinia lacus blandit, eu interdum libero semper. Suspendisse potenti. Sed a vulputate neque. Fusce in aliquam arcu, id tempor velit. Morbi aliquet erat nec magna eleifend efficitur. Sed vitae eros mollis, interdum enim aliquet, maximus est. 
```
From: gemini://alex.flounder.online/misc/blockquote.gmi

Lagrange renders the first three lines as though the are one single multi-line quote, instead of three separate lines.

According to the spec, at least as I understand it, subsequent quote lines should not be rendered as part of the same block. This is a little unintuitive, but most clients I tested don't seem to do this, Lagrange appears to be an exception. See this mailing list conversation:

https://lists.orbitalfox.eu/archives/gemini/2021/004693.html

Thanks for Lagrange!

**Comment by skyjake** [comment]
The current behavior of zero-spacing consecutive quotes in Lagrange is to have these behave consistently:
```
First line.
Second line.
Third line.

> First line.
> Second line.
> Third line.
```
Which looks like this:
<img width="189" alt="Screen Shot 2021-01-04 at 7 30 33" src="https://user-images.githubusercontent.com/1225742/103504054-fccb3400-4e5e-11eb-9efd-68c094196bfc.png">
Or:
<img width="170" alt="Screen Shot 2021-01-04 at 7 33 29" src="https://user-images.githubusercontent.com/1225742/103504119-28e6b500-4e5f-11eb-871b-1df7603be522.png">

In the first case , with the **"** symbol, it would look ugly to repeat the icon on each line.

**Comment by skyjake** [comment]
The current implementation is a bit too eager with this, though, because it ignores empty lines when checking for consecutive quotes. It could be argued that if there is an empty line between the quotes, they are intended to be separate.

```
> "Hey, don't forget to water your plant!"
> - DrSkrzyk, 2021-01-03

> "harvest -> epilog -> epilog2 -epilog3"
> - triffid, 2021-01-01

> "Just joining in with GemiNaut which now supports client certs - hi everyone"
> - marmaladefoo, 2021-01-01
```

I'll change this so that empty lines will cause the quote icon to reappear. Not sure if there's anything else that should be changed here.

**Comment by skyjake** [comment]
Closing this as changes included in v1.0.3.
